src/patternfly/components/Radio/radio.scss

```diff
@@ -6,6 +6,7 @@
   --pf-c-radio__label--FontSize: var(--pf-global--FontSize--md);
   --pf-c-radio__label--LineHeight: var(--pf-global--LineHeight--sm);
   --pf-c-radio__input--MarginTop: #{pf-size-prem(-3px)};
+  --pf-c-radio__input--MarginLeft: #{pf-size-prem(1px)};
```

Comment:
since we don't usually add a margin to elements, should you leave a note here that this fixes the OSX bug ? so devs know what this margin left is for. If they don't see the issue I think they will be confused